setup.py

```diff
@@ -30,6 +30,7 @@ def read(*names, **kwargs):
     'Jinja2',
     'pyyaml',
     'ploomber>=0.14.6',
+    'ploomber_core',
```

Comment:
```suggestion
    'ploomber-core',
```
that's the right name of the package. I was surprised that this works as well! TIL pip replaces the `_` with `-`, that's why it didn't break